### ISSUE TITLE
Suggestion: Include example of code without gulp-watch-sequence

### ISSUE THREAD

**Issue by isochronous** [bholloway/gulp-watch-sequence#3]
While I *think* I understand what this library does, it would be a lot clearer if your code example was more of a "before" and "after" thing, and preferably showed ALL the code that's referenced, like the 'html', 'reload', and 'js' tasks (even if they were just empty dummy tasks, at least seeing them being defined somewhere would be less confusing than string references to code that doesn't exist).  Something like "this is how you'd traditionally do this without gulp-watch-sequence, but here are the issues that the traditional implementation has.  Now here's how you do it with gulp-watch-sequence, and those issues have been solved."

Basically, I keep coming back to look at this library because I have the feeling that it would be useful to me, but the included code example doesn't really tell me much.
